specification/ml/_types/TrainedModel.ts

```diff
@@ -229,11 +319,31 @@ export class TrainedModelAllocationRoutingTable {
   routing_state: RoutingState
 }
 
+export class TrainedModelDeploymentAllocationStatus {
+  /** The current number of nodes where the model is allocated. */
+  allocation_count: integer
+  /** The sum of `error_count` for all nodes in the deployment. */
+  error_count: integer
```

Comment:
This has 3 fields: `allocation_count`, `target_allocation_count` & `state`.


`error_count`, `rejected_execution_count` and `timeout_count` are in `TrainedModelDeploymentStats`

Reply:
I think this implies that the docs are wrong in https://elasticsearch_82705.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/get-trained-models-stats.html, since I see `error_count` etc nested in `allocation_status` there.

Reply:
Yes sorry the `====` was in the wrong place I've updated the PR in

https://github.com/elastic/elasticsearch/pull/82705/commits/e32ab2a54f3ce0922ddc79f5672dfe8d13d4754e

Reply:
Thanks! Fixed in 1779cd7b7a327f14cda368fd5d0f74b3d101fcbb